include/certifier_framework.i

```diff
@@ -13,9 +13,19 @@
 %module certifier_framework
 %include "std_string.i"
 
-// Needed to invoke init_policy_key() using Python bytestream arg.
+// Needed to invoke init_policy_key() and other interfaces that take
+// (byte *, int) supplied by Python bytestream arg.
 %include <pybuffer.i>
-%pybuffer_binary(byte* asn1_cert, int asn1_cert_size)   // cc_trust_data()->init_policy_key()
+
+%pybuffer_binary(const byte *serialized_attest_key, int attest_key_size);
+%pybuffer_binary(const byte *measurement, int measurement_size);
+%pybuffer_binary(const byte *serialized_attest_endorsement, int attest_key_signed_claim_size);
```

Comment:
These 3 directives are to marshall the parameters of `initialize_simulated_enclave()` as byte-stream arguments.